rendering/fbx.ksy

```diff
@@ -0,0 +1,148 @@
+meta:
+  id: fbx
+  title: FBX 3D model interchange format
+  file-extension: fbx
+  encoding: UTF-8
+  endian: le
+  license: MIT
+
```

Comment:
Lacks `eoc` and `doc-ref`. Also may make sense to provide a link to a publicly-available sample file.

Reply:
`eoc`?

Reply:
`doc`. A typo.

Reply:
Ah. Well, they don't document the format so there was nothing to put in I guess. Blender had an attempt at documenting the format - is it okay to put in a third-party document describing the format?

Reply:
They don't but you can put there some words from you, at least a brief summary of the format is needed: what its purpose is, what it is it used for, + anything you consider non-obvious.

Reply:
Got it.